tools/schemacode/bidsschematools/validator.py

```diff
@@ -42,17 +42,11 @@ def _get_paths(
     * Deduplicate paths (if input dirs are subsets of other input dirs), might best be done at the
         very end.
     """
-    exclude_subdirs = [
-        rf"{os.sep}.dandi",
-        rf"{os.sep}.datalad",
-        rf"{os.sep}.git",
-    ]
     # `.bidsignore` is not, in fact, a BIDS file, as per:
     # https://github.com/bids-standard/bids-specification/issues/980
+    # Perhaps this should be parameterized for downstream flexibility and not having to keep track
+    # of downstream nuisance files here.
```

Comment:
I think this is probably the right move in the long run. I think we might want to make the interface `validate(..., ignore=ignore_func)` where the first step when looking at a file is `if ignore_func(rel_path): continue`.

But that doesn't need to be done here.